src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (input) => calculator.result + input,
+
+    subtract: (input) => calculator.result - input,
```

Comment:
The 'subtract' method should modify the 'result' property instead of just returning the subtraction. You may want to refactor it to 'calculator.result -= input';